bitmap_amd64.go

```diff
@@ -75,6 +75,9 @@ func (dst *Bitmap) Or(other Bitmap, extra ...Bitmap) {
 	if max == 0 {
 		return
 	}
+	if len(other) == 0 {
```

Comment:
While this technically works for your use case, it would probably be better to avoid the allocation altogether for what is a no-op (`a == (a|0)`).
```
if cap(other) == 0 {
    return
}
```

Reply:
+1